sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/platform/NotificationsModel.kt

```diff
@@ -77,8 +77,8 @@ object NotificationsModel {
             sessions.forEach { session ->
                 if (session.feedbackRating == null) {
                     val feedbackNotificationTime = session.endsAt.toLongMillis() + Durations.TEN_MINS_MILLIS
-                    ServiceRegistry.notificationsApi.createLocalNotification("How was the session?",
-                            " Leave feedback for " + session.title,
+                    ServiceRegistry.notificationsApi.createLocalNotification("Feedback Time!",
+                            "Your Feedback is Requested",
                             feedbackNotificationTime,
                             //Not great. Possible to clash, although super unlikely
                             session.id.hashCode(),
```

Comment:
I think since the feedback notification is no longer linked to a particular session. We can just have a single notification id and if we are already showing the notification, we don't need to update it

Reply:
This could be done in the NotificationsApiImpl, although we'd have to change the NotificationsApi so that there is an option to update notifications for the reminders.

Reply:
Updated to use shared FeedbackId